jquery.orgchart.css

```diff
@@ -79,21 +79,27 @@ div.orgChart.interactive div.hasChildren {
 div.orgChart div.node.hover {
     background-color      : white;
 }
+/* rtl */
+div.orgChart.ltr{ direction: ltr; }
+div.orgChart.rtl{ direction: rtl; }
 
-div.orgChart div.adjunct.node {
+div.orgChart.ltr div.adjunct.node , div.orgChart.rtl div.adjunct.node {
```

Comment:
These .ltr and .rtl selectors seem not necessary to me here. They have the same behaviour here.